### PR TITLE
add JetBrain Open Source Community Support

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

**Review by qwerty287 — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -85,6 +85,12 @@ In order to use the new photos email notification you will need to have configur
 
 Take a look at the [Documentation](https://lycheeorg.github.io/docs/), particularly the [FAQ](https://lycheeorg.github.io/docs/faq.html) if you have problems. Discovered a bug? Please create an issue [here](https://github.com/LycheeOrg/Lychee/issues) on GitHub! You can also contact us directly on [gitter (login with your github account). &#187;](https://gitter.im/LycheeOrg/Lobby)
 
+## Open Source Community Support
+
+<img src="https://resources.jetbrains.com/storage/products/company/brand/logos/PhpStorm_icon.png" alt="PhpStorm" width="50"/>
+
+We would like to thank Jetbrains for supporting with their [Open Source Development - Community Support][jetbrains-opensource] program.
```
Comment (nagmat84) [inline readme.md]:
```suggestion
We would like to thank Jetbrains for supporting us with their [Open Source Development - Community Support][jetbrains-opensource] program.
```

**Review by nagmat84 — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -85,6 +85,12 @@ In order to use the new photos email notification you will need to have configur
 
 Take a look at the [Documentation](https://lycheeorg.github.io/docs/), particularly the [FAQ](https://lycheeorg.github.io/docs/faq.html) if you have problems. Discovered a bug? Please create an issue [here](https://github.com/LycheeOrg/Lychee/issues) on GitHub! You can also contact us directly on [gitter (login with your github account). &#187;](https://gitter.im/LycheeOrg/Lobby)
 
+## Open Source Community Support
+
+<img src="https://resources.jetbrains.com/storage/products/company/brand/logos/PhpStorm_icon.png" alt="PhpStorm" width="50"/>
```
Comment (d7415) [inline readme.md:90]: This upsets me on several levels. Mostly downloading a 200kb png for a 50px logo, which looks fairly clunky anyway.

File: readme.md [hunk 3]
```diff
@@ -99,3 +105,6 @@ Take a look at the [Documentation](https://lycheeorg.github.io/docs/), particula
 [frontend-shield]: https://img.shields.io/badge/-Lychee--Front-informational.svg?logo=github
 [docker-shield]: https://img.shields.io/badge/-Lychee--Docker-informational.svg?logo=github
 [gitter-shield]: https://img.shields.io/gitter/room/LycheeOrg/Lobby.svg?logo=gitter
+[phpStorm]: https://resources.jetbrains.com/storage/products/company/brand/logos/PhpStorm_icon.png
```
Comment (d7415) [inline readme.md]: With the HTML img tag (for resizing) this isn't being used.
```suggestion
```

File: readme.md (hunk 2 above)
Comment (ildyria) [reply readme.md:90]: Agreed. :( That's their logo though and they don't provide lower res (but do provide svg). :(

**Pull request merged**